package.dhall

```diff
@@ -1 +1,4 @@
-{ map = ./map/package.dhall }
+{ Map = ./map/package.dhall
+, Kubernetes = ./kubernetes/package.dhall
+, Log = ./log/package.dhall
+}
```

Comment:
v.s.

Reply:
See above

Reply:
so here because the folder is called `map` I would expect to be able to call `UtilityLibrary.map` not `UtilityLibrary.Map`. As `map/package.dhall` is a record (albeit one of types) I would suggest changing this to `{ map = ./map/package.dhall, kubernetes = ... }`

Reply:
I am not a fan, but I will change it.